### PR TITLE
Add new Racket translation to spanish

### PR DESCRIPTION
**Pull request by carlochess**

### CODE REVIEW

File: es-es/racket-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+  - ["Keyan Zhang", "https://github.com/keyanzhang"]
+translators:
+    - ["Carlos Roman", "https://github.com/carlochess"]
+lang: es-es
+---
+Racket es un lenguaje de propósito general, multiparadigma que hace parte de la familia Lisp/Scheme
+Racket is a general purpose, multi-paradigm programming language in the Lisp/Scheme family.
+
+
+La retroalimentación de bienvenida, puedes encontrarme en [@th3rac25](http://twitter.com/th3rac25) or th3rac25 [at] [google's email service]
```
Comment (fitojb) [inline es-es/racket-es.html.markdown]: > La retroalimentación de bienvenida

This does not make sense to me. A better translation would be “Agradecemos tus opiniones”.

Comment (carlochess) [inline es-es/racket-es.html.markdown]: Tenes razón, escribí  "de" cuando debia ser "es". 

La retroalimentación es bienvenida...

Comment (vendethiel) [inline es-es/racket-es.html.markdown]: @carlochess can you fix it please? thanks :)

**Comment by vendethiel** [comment]
Going to close this one in the meantime. Please ping me if you find some time :-).

**Comment by vendethiel** [comment]
Thanks. *please* remember to comment as well when you push commits, as github doesn't send notifications...

**Pull request merged**